src/object/C4Object.h

```diff
@@ -101,6 +101,7 @@ class C4Object: public C4PropListNumbered
 	void Splash();
 	void RemoveSolidMask(bool fBackupAttachment); // Remove solid mask data, if existing
 	void MovementDigFreeTargetArea(); // Dig the area free, according to action data
+	inline void FinishRemoval(bool exit_contents);
```

Comment:
Why inline? In this case it's not needed to be able to include it (as it's defined in the .cpp file). And compilers will decide themselves anyway what to actually inline and what not.
See https://stackoverflow.com/questions/29796264/is-there-still-a-use-for-inline

Reply:
Thanks for the article! So, the purpose was to separate the parts "Start removing the object and do all the callbacks" (=AssignRemoval) and "Actually remove the object" (=FinishRemoval). Making it inline had the intention of telling the compiler to not make it an actual separate function.

Reply:
> Why inline? In this case it's not needed to be able to include it (as it's defined in the .cpp file).
It wouldn't be required even if it was defined here because method definitions inside the class definition always have inline linkage.